RockLib.UniversalMemberAccessor.Tests/TypeCreator/Define.cs

```diff
@@ -117,17 +117,17 @@ public static MemberDefinition Method(string name, bool isStatic = false, Visibi
                     }
                 }
 
-                il.Emit(OpCodes.Ldstr, "void " + tb.Name + "." + name + "(" + string.Join(", ", parameters.Select(p => p.Type.Name)) + ")");
-                il.Emit(OpCodes.Call, typeof(Console).GetMethod("WriteLine", new[] { typeof(string) }));
+                il.Emit(OpCodes.Ldstr, "void " + tb.Name + "." + name + "(" + string.Join(", ", parameters.Select(p => p.Type?.Name)) + ")");
```

Comment:
Consider changing this to string interpolation.

Reply:
Updated to string interpolation.